client/my-sites/github-deployments/deployment-run-logs/deployments-run-item.tsx

```diff
@@ -65,7 +71,19 @@ export const DeploymentsRunItem = ( { run }: DeploymentsListItemProps ) => {
 					<DeploymentDuration run={ run } />
 				</td>
 				<td>
-					<Button plain onClick={ handleToggleExpanded }>
+					<Button
+						plain
+						onClick={ () => {
+							if ( ! expanded ) {
+								dispatch(
+									recordTracksEvent( 'calypso_hosting_github_log_entry_expanded', {
+										row_number: rowNumber,
+									} )
+								);
```

Comment:
The same thing here.

Reply:
That's how I started, but it fires the event twice when inlined.